src/eachobs.jl

```diff
@@ -1,21 +1,29 @@
 """
     eachobs(data, buffer=false, batchsize=-1, partial=true)
 
-Return an iterator over the observations in `data`. 
+Return an iterator over the observations in `data`.
 
 # Arguments
 
-- `data`. The data to be iterated over. The data type has to implement 
-  [`numobs`](@ref) and [`getobs`](@ref). 
-- `buffer`. If `buffer=true` and supported by the type of `data`, 
+- `data`. The data to be iterated over. The data type has to implement
+  [`numobs`](@ref) and [`getobs`](@ref).
```

Comment:
```suggestion
- `data`. The data to be iterated over. The data type has to be supported by
  [`numobs`](@ref) and [`getobs`](@ref).
```
since `numobs` and `getobs` can support a datatype (through indexing) even if specialized implementations are not given